kurtosis-module/impl/cl_client_network/lighthouse/lighthouse_launcher.go

```diff
@@ -0,0 +1,216 @@
+			return nil, stacktrace.Propagate(
+				err,
+				"An error occurred copying the config data directory on the module, '%v', into the service container, '%v'",
+				launcher.configDataDirpathOnModuleContainer,
+				destConfigDataDirpathOnModule,
+			)
+		}
+
+		elClientRpcUrls := []string{}
+		for rpcSocketStr := range elClientRpcSockets {
+			rpcUrlStr := fmt.Sprintf("http://%v", rpcSocketStr)
+			elClientRpcUrls = append(elClientRpcUrls, rpcUrlStr)
+		}
+		elClientRpcUrlsStr := strings.Join(elClientRpcUrls, ",")
+
+		configDataDirpathOnService := configDataDirpathOnServiceSharedPath.GetAbsPathOnServiceContainer()
+		// NOTE: If connecting to the merge devnet remotely we DON'T want the following flags; when they're not set, the node's external IP address is auto-detected
+		//  from the peers it communicates with but when they're set they basically say "override the autodetection and
+		//  use what I specify instead." This requires having a know external IP address and port, which we definitely won't
+		//  have with a network running in Kurtosis.
+		//    "--disable-enr-auto-update",
+		//    "--enr-address=" + externalIpAddress,
+		//    fmt.Sprintf("--enr-udp-port=%v", discoveryPortNum),
+		//    fmt.Sprintf("--enr-tcp-port=%v", discoveryPortNum),
+		cmdArgs := []string{
+			"lighthouse",
+			"--debug-level=info",
+			"--datadir=" + consensusDataDirpathOnServiceContainer,
+			"--testnet-dir=" + configDataDirpathOnService,
+			"bn",
+			"--eth1",
+			// vvvvvvvvvvvvvvvvvvv REMOVE THESE WHEN CONNECTING TO EXTERNAL NET vvvvvvvvvvvvvvvvvvvvv
+			"--disable-enr-auto-update",
+			"--enr-address=" + privateIpAddr,
+			fmt.Sprintf("--enr-udp-port=%v", discoveryPortNum),
+			fmt.Sprintf("--enr-tcp-port=%v", discoveryPortNum),
+			// ^^^^^^^^^^^^^^^^^^^ REMOVE THESE WHEN CONNECTING TO EXTERNAL NET ^^^^^^^^^^^^^^^^^^^^^
+			"--listen-address=0.0.0.0",
+			fmt.Sprintf("--port=%v", discoveryPortNum), // NOTE: Remove for connecting to external net!
+			"--http",
+			"--http-address=0.0.0.0",
+			fmt.Sprintf("--http-port=%v", httpPortNum),
+			"--merge",
+			"--http-allow-sync-stalled",
+			"--disable-packet-filter",
+			"--execution-endpoints=" + elClientRpcUrlsStr,
+			"--eth1-endpoints=" + elClientRpcUrlsStr,
+			fmt.Sprintf("--terminal-total-difficulty-override=%v", totalTerminalDiffulty),
+		}
+		if bootNodeEnr != bootnodeEnrStrForStartingBootnode {
+			cmdArgs = append(cmdArgs, "--boot-nodes=" + bootNodeEnr)
+		}
+
+		containerConfig := services.NewContainerConfigBuilder(
+			imageName,
+		).WithUsedPorts(
+			usedPorts,
+		).WithCmdOverride(
+			cmdArgs,
+		).Build()
+		return containerConfig, nil
+	}
+}
+
+func waitForAvailability(restClient *cl_client_rest_client.CLClientRESTClient) error {
+	for i := 0; i < maxNumHealthcheckRetries; i++ {
+		_, err := restClient.GetHealth()
+		if err == nil {
+			// TODO check the healthstatus???
```

Comment:
This comment indicates that probably is missing a condition to check the node status

Reply:
Yeah so, there's a weird technicality here - the API can return an actual repsonse (indicating the hTTP server is available) but that response's status code can mean several things: https://ethereum.github.io/beacon-APIs/#/Node/getHealth
I suspect that the node will return 503 at first and then, after a little while, move to 206, and then to 200. But, we don't want to wait for the node to become fully available before we start the next ones (just want the REST API to be available to guarantee the node actually started) which is why I ignore this for now.

Reply:
ok, makes sense